openmcyclus/depletion.py

```diff
@@ -24,34 +25,28 @@ def __init__(self, chain_file: str,
             the number given is in units of months. This value is
             multiplied by 30 to give a timestep in days.
         power: float
-            power output of the reactor, assumed in MWe.
-        conversion_factor: float
-            conversion factor to go from MWe to MWth. Default value of 3,
-            representing a thermal efficiency of 1/3
-        '''
-        self.chain_file = chain_file
-        self.timesteps = timesteps
-        self.power = power
-        self.conversion_factor = conversion_factor
-
-    def read_materials(self, path):
-        '''
-        Read in the materials file for OpenMC
+            power output of the reactor, assumed in MWth.
+        path: str
+            relative path to micro xs and materials files
 
-        Parameters:
+        Attributes:
         -----------
+        chain_file: str
+            file name for decay chain data
+        timesteps: int
+            number of time steps to perform depletion. It is assumed that
+            the number given is in units of months. This value is
+            multiplied by 30 to give a timestep in days.
+        power: float
+            power output of the reactor, assumed in MWth.
         path: str
-            path to the materials.xml file
+            relative path to micro xs and materials files
```

Comment:
```suggestion
            relative path to the micro_xs.csv and materials.xml files
```
Or something similar.